### PR TITLE
Mainnet release

### PR DESCRIPTION
**Pull request by aulneau**

### CODE REVIEW

File: src/common/constants.ts [hunk 1]
```diff
@@ -19,8 +19,8 @@ export const DEFAULT_MAINNET_SERVER = process.env.NEXT_PUBLIC_MAINNET_API_SERVER
 
 export const MAINNET_ENABLED = process.env.NEXT_PUBLIC_MAINNET_ENABLED === 'true';
 
-export const NETWORK_LIST_COOKIE = 'network-list';
-export const NETWORK_CURRENT_INDEX_COOKIE = 'network-current';
+export const NETWORK_LIST_COOKIE = 'STAGING__NETWORK_LIST';
+export const NETWORK_CURRENT_INDEX_COOKIE = 'STAGING__NETWORK_CURRENT_INDEX';
```
Comment (hstove) [inline src/common/constants.ts:23]: Why this change?

File: src/components/metaverse-bg.tsx [hunk 2]
```diff
@@ -66,51 +66,59 @@ const Grain: React.FC<BoxProps> = props => (
...
+        width="100%"
+        top={0}
+        height={height}
+        overflow="hidden"
+        transition={transition}
+      >
+        <GlobalStyles />
+        <Grain opacity={0.45} />
+        <Grain />
+        <Video />
```
Comment (hstove) [inline src/components/metaverse-bg.tsx:109]: How does this stuff work? Is there some kind of logic for which to show?

**Review by kyranjamie — APPROVED** [review]
Production-ready blockchain explorer

File: src/components/sandbox/views/signed-out-view.tsx [hunk 3]
```diff
@@ -33,7 +35,15 @@ const ItemCaption = (props: BoxProps) => (
 
 export const SignedOutView: React.FC<any> = () => {
   const { doOpenAuth } = useConnect();
+  const [installed, setInstallStatus] = useState(false);
 
+  const extInstalled = IS_BROWSER && typeof window.BlockstackProvider !== 'undefined';
```
Comment (hstove) [inline src/components/sandbox/views/signed-out-view.tsx:40]: FYI I literally just changed it to `StacksProvider` but I still have `BlockstackProvider` for compatibility.

**Review by hstove — APPROVED** [review]

**Review by hstove — APPROVED** [review]

File: src/common/constants.ts (hunk 1 above)
Comment (aulneau) [reply src/common/constants.ts:23]: I will change it before release, but there was some legacy saved data that was interfering with some peoples experience

File: src/components/metaverse-bg.tsx (hunk 2 above)
Comment (aulneau) [reply src/components/metaverse-bg.tsx:109]: They are all layered. Two grains so we can adjust the amount of grain as needed, and the video below, and then placeholder images below that for when video isn't/doesn't load.
